packages/nodes-base/nodes/Adalo/Adalo.node.ts

```diff
@@ -0,0 +1,188 @@
+import {
+	IExecuteFunctions,
+} from 'n8n-core';
+
+import {
+	IDataObject,
+	INodeExecutionData,
+	INodeType,
+	INodeTypeDescription,
+} from 'n8n-workflow';
+
+
+import {FieldsUiValues, Operation} from './types';
+import {adaloApiRequest} from './GenericFunctions';
+import {operationFields} from './OperationDescription';
+
+/**
+ * The Adalo API Key is available on a per-app basis, providing a CRUD REST API
+ * to interact with any collection defined in the application
+ *
+ * @docs https://help.adalo.com/integrations/the-adalo-api/collections
+ */
+
+export class Adalo implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'Adalo',
+		name: 'adalo',
+		icon: 'file:adalo.png',
+		group: ['transform'],
+		version: 1,
+		description: 'Consume the Adalo API',
+		defaults: {
+			name: 'Adalo',
+		},
+		inputs: ['main'],
+		outputs: ['main'],
+		credentials: [
+			{
+				name: 'adaloApi',
+				required: true,
+			},
+		],
+		properties: [
+			{
+				displayName: 'Collection ID',
+				name: 'collectionId',
```

Comment:
Does the API allow to fetch the collection IDs? If so, we should load them for the user so he can easily pick them.

Reply:
The API Is very limited. There's no way (I haven't found one, at least - looked in the forums as well) to get collection ids.